Assets/UI/Civilopedia/civilopediasupport.lua

```diff
@@ -847,6 +847,13 @@ function NavigateTo(SectionId, PageId)
   end
 end
 
+function OnToggleCivilopedia()
+    if(ContextPtr:IsHidden()) then 
+      OnOpenCivilopedia();
+    else
+		OnClose();
+    end
+end
```

Comment:
still not right here
```
function OnToggleCivilopedia()
  if(ContextPtr:IsHidden()) then 
    OnOpenCivilopedia();
  else
    OnClose();
  end
end
```

Reply:
***Indentation means, how you format the nested parts of code.***
Some do it with tabs, some with spaces. Each language, and more often each project follows its own rules, whether to use tabs or spaces. Some use 2 spaces for indentation, some 4 and there can be quite some religious argument about this issue.
**Why is this important:** Code gets increasingly illegible if different coding styles are mixed. To get a clear image of what is going on, you need to have people follow the guidelines. Even more so when having so many files and contributors. And I think Azurency is so persistent in asking for correction, because he does not want a culture of sloppiness to develop.
**For a practical hint:** set your editor to insert 2 space characters instead of tabs when pressing the tab key, set it to show tabs e.g. 8 spaces to easily spot them, and after copying text replace all tabs by 2 spaces to fit into the scheme.